heartcheck-newrelic.gemspec

```diff
@@ -14,8 +14,6 @@ Gem::Specification.new do |spec|
   spec.homepage = 'https://github.com/locaweb/heartcheck-newrelic'
 
   spec.files = `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^spec/}) }
```

Comment:
Can we reject bin too? This way we can ensure we're not injecting `bin/rspec` again

Reply:
Sure!

Seems that although I've removed the executables configuration, it still adds the bin folder, thanks to the `git ls-files` command...

I opted to add `bin` to the reject as you said... The other option was to change the `git ls-files`, but there would be lots of changes, and increased chances of breaking something else. Take a look:

WDYT?

Reply:
looks great!